.travis.yml

```diff
@@ -95,10 +95,30 @@ install:
     - mkdir build && cd build
     - cmake .. && make -j3 && make test ARGS=-j3 && sudo make install
 
+    # Install google benchmark
```

Comment:
Good job on installing Google benchmark on Travis. 

Could you or @guacke make sure this installation is also cached? Do we do this in a separate PR after this one, or here?

Reply:
We'll merge this PR and make an issue later.